packages/fast-components-styles-msft/src/utilities/important.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Sets a css value to be '!important'.
+ */
+export function importantValue(value: string): string {
```

Comment:
Since this is intended for style sheets, and most style sheet functions are DesignSystemResolvers, can you change this to `value: string | DesignSystemResolver<string>` and then call `checkDesignSystemResolver(value)`?

Reply:
updated. thanks Brian